Doc/howto/pyporting.rst

```diff
@@ -1,49 +1,47 @@
 .. _pyporting-howto:
 
-*********************************
-Porting Python 2 Code to Python 3
-*********************************
+*************************************
+How to port Python 2 Code to Python 3
```

Comment:
Not sure why we need to change the title

Reply:
Every genuine how-to guide should have a title "How to <xxx>" to signal clearly what its purpose and scope is.

In menus, the title can be shortened to "<xxx>". Eventually it means you can have a menu that looks like:

```
How to
    Install
    Deploy
    Scale
```

etc. So the same clarity of purpose is recapitulated there.

An exception is when it might be clunky to use "how to" - for example I think that "Troubleshooting" is firstly rather less hideous than "How to troubleshoot" and secondly already implies that it's about taking action (in a way that "porting" does not).

"Porting Python 2 Code to Python 3" is ambiguous. Maybe it's not a how-to guide - maybe it's what I should read to find out why I should do that.

If you apply this logic to other documents in that HOWTO section, it becomes clear that many of them cannot be given "How to..." titles - because their content is definitely not how-to content. An example is the functional programming document, which is one that I think should find a different home, for that reason. 

Paying close attention to titles is helpful for the clues it gives us about documentation, and sets down good markers for future work.